lib/util/fs/index.ts

```diff
@@ -133,3 +135,21 @@ export async function findLocalSiblingOrParent(
 
   return null;
 }
+
+/**
+ * Get files by name from directory
+ */
+export function readLocalDirectory(path: string): Promise<string[]> {
+  return fs.readdir(path);
```

Comment:
```suggestion
  const localPath = join(localDir, path);
  return fs.readdir(localPath);
```

Local function only work on repo relative files. Normal fs function should added to `./proxies.ts` and can be path through from `fs-extra`